### PR TITLE
chore: update scripts to initialize with project_name

### PR DESCRIPTION
**Pull request by migmartri**
Update labs code to get ready for this change https://github.com/chainloop-dev/chainloop/issues/1375

It adds the requirement of doing `att init` indicating the project name. 

Note that I also updated the onboarding project to return the project name too.

### CODE REVIEW

File: .github/workflows/chainloop_github_release.yml [hunk 1]
```diff
@@ -30,10 +30,10 @@ on:
 jobs:
   onboard_workflow:
     name: Onboard Chainloop Workflow
-    uses: chainloop-dev/labs/.github/workflows/chainloop_onboard.yml@4173e015dbd5dc2a8802555c268da63d57bbe576
```
Comment (migmartri) [inline .github/workflows/chainloop_github_release.yml:33]: will update once we release the change

File: .github/workflows/chainloop_github_release.yml [hunk 2]
```diff
@@ -30,10 +30,10 @@ on:
...
+    uses: chainloop-dev/labs/.github/workflows/chainloop_onboard.yml@main
     if: github.event_name == 'release' && github.event.action == 'published'
     with:
-      project: ${{ inputs.workflow_project }}
```
Comment (migmartri) [inline .github/workflows/chainloop_github_release.yml:36]: I am not sure how this ever worked. Did I miss anything cc/ @javirln

File: tools/c8l [hunk 3]
```diff
@@ -676,7 +676,11 @@ chainloop_attestation_init() {
   if [ -n "${CHAINLOOP_WORKFLOW_NAME}" ]; then
     WF_NAME_VALUE="--workflow-name ${CHAINLOOP_WORKFLOW_NAME}"
   fi
-  r=$(chainloop attestation init -f --remote-state --output json $CR_VALUE $WF_NAME_VALUE)
+  PROJECT_NAME_VALUE=""
```
Comment (migmartri) [inline tools/c8l:679]: @danlishka I might need your help here. I am trying to figure out how the push works.

I think it uses some kind of cache, but I guess that now we will need to update the cache to also take into account the project name, but I am struggling to find that. No rush

**Comment by migmartri** [comment]
Coverted to draft until I figure out how to make the `push` work with the cache.

File: tools/c8l (hunk 3 above)
Comment (migmartri) [reply tools/c8l:679]: ahh no, the cache is unique within the run, ok.

I am wondering then why we are passing the workflow name here

https://github.com/chainloop-dev/chainloop/blob/f215274ee36f6f0cb7010eadaeafbb60b394265c/.github/workflows/docs_deploy.yml#L96

**Review by jiparis — APPROVED** [review]

**Pull request merged**